src/azure-cli/setup.py

```diff
@@ -55,75 +55,75 @@
     'azure-batch~=15.0.0b1',
     'azure-cli-core=={}'.format(VERSION),
     'azure-cosmos~=3.0,>=3.0.2',
-    'azure-data-tables==12.4.0',
+    'azure-data-tables~=12.4.0',
     'azure-datalake-store~=1.0.0a0',
-    'azure-keyvault-administration==4.4.0b2',
-    'azure-keyvault-certificates==4.7.0',
-    'azure-keyvault-keys==4.11.0b1',
-    'azure-keyvault-secrets==4.7.0',
-    'azure-mgmt-advisor==9.0.0',
-    'azure-mgmt-apimanagement==4.0.0',
-    'azure-mgmt-appconfiguration==3.1.0',
-    'azure-mgmt-appcontainers==2.0.0',
+    'azure-keyvault-administration~=4.4.0b2',
```

Comment:
There is now a stable `4.4.0` (and even a `4.5.0`) release.
- https://pypi.org/project/azure-keyvault-administration/4.4.0/
- https://pypi.org/project/azure-keyvault-administration/4.5.0/